backend/src/docker-compose.yml

```diff
@@ -47,13 +47,15 @@ services:
     ports:
       - 5432:5432
     volumes:
-      - /postgres_data:/var/lib/postgresql/data
+      - ./postgres_data:/var/lib/postgresql/data
       
   frontend:
     image:  ${DOCKER_REGISTRY-}stamacasa
     build:
         context: ../../frontend
         dockerfile: Dockerfile
+    environment: 
+      - REACT_APP_IDP_URL: 'http://localhost:5001'
```

Comment:
I don't believe the docker file is used in development... If my statement is true, could we switch this to `http://stam-acasa-dev-identitysrv-1419121392.eu-west-1.elb.amazonaws.com`, as used in production?

Please also set `API_URL` to `http://stam-acasa-dev-api-151370545.eu-west-1.elb.amazonaws.com` or `http://localhost:5008` for development